compiler/rustc_metadata/src/creader.rs

```diff
@@ -68,11 +68,11 @@ impl std::fmt::Debug for CStore {
 pub struct CrateLoader<'a> {
     // Immutable configuration.
     sess: &'a Session,
-    metadata_loader: Box<MetadataLoaderDyn>,
+    metadata_loader: &'a MetadataLoaderDyn,
     local_crate_name: Symbol,
-    // Mutable output.
```

Comment:
Could you keep this comment and the separation into input and output fields?

Reply:
done